python_modules/dagster/dagster/_core/storage/cloud_storage_compute_log_manager.py

```diff
@@ -112,7 +112,10 @@ def _on_capture_complete(self, log_key: Sequence[str]):
         self.upload_to_cloud_storage(log_key, ComputeIOType.STDERR)
 
     def is_capture_complete(self, log_key: Sequence[str]) -> bool:
-        return self.local_manager.is_capture_complete(log_key)
+        if self.local_manager.is_capture_complete(log_key):
+            return True
+        # check remote storage
+        return self.cloud_storage_has_logs(log_key, ComputeIOType.STDERR)
```

Comment:
why ERR? we can assume it also has out?

Reply:
well, for op-based log capture, err will always be populated, not necessarily out.  But you're right we should check both.